client/my-sites/plans/jetpack-plans/product-lightbox/index.tsx

```diff
@@ -108,7 +118,15 @@ const ProductLightbox: React.FC< Props > = ( {
 		} ) );
 	}, [ product.productSlug ] );
 
-	const shouldShowOptions = variantOptions.length > 1;
+	const showVariantsWithoutRadioButtons = [
```

Comment:
This is a bit of a nitpick (and I'm struggling to think of a better option) but this name and the `showPlansOneBelowTheOther` seem a bit too tied to the design.

Maybe it's something for a follow-up but all the conditional checks might mean that we could break this into multiple components and have two styles for the product variants.

Reply:
I agree with the suggestion.

Reply:
I have made it a component, but the variable name I am not convinced, but I can definitely think of a better name and update it in the future @pablinos.